### PR TITLE
[KOGITO-8249] Updating timeouts guide

### PR DESCRIPTION
**Pull request by fjtirado**
**JIRA:**
https://issues.redhat.com/browse/KOGITO-8249

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 1]
```diff
@@ -7,26 +7,37 @@
 :jobs_service_image_url: https://quay.io/repository/kiegroup/kogito-jobs-service-allinone
 :jobs_service_image_usage_url: https://github.com/kiegroup/kogito-images#jobs-services-all-in-one
 
+
+Serverless Workflow specification defines a wide amount of possible link:{spec_doc_url}#workflow-timeout-definition[timeout configurations]. 
+Among them, {{product_name}} currently supports workflow and event timeouts.
```
Comment (ricardozanini) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:12]:
```suggestion
Among them, {product_name} currently supports workflow and event timeouts. 
```

**Review by ricardozanini — COMMENTED** [review]
Thanks! 

Can you also update this section? 

https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/getting-started/cncf-serverless-workflow-specification-support.html#timeouts

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 2]
```diff
@@ -7,26 +7,37 @@
...
+You can set up the maximum amount of time a workflow might be running before being cancelled. This is configured in the header section of the flow, by using link:{spec_doc_url}#workflowexectimeout-definition[WorkflowExecTimeout definition]. Only `duration` property is currently implemented. 
+Once cancelled, the workflow is considered finished and won't be accessible though a GET request anymore. So it behaves as interrupt was `true` by default. 
+
+For example, in order to cancel the workflow after an hour of execution, you might add the following snippet. 
+.Example of workflow timeout
+[source,json]
+----
+"timeouts": {
+    "workflowExecTimeout": "PT1H"
+  }
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:29]:
```suggestion
"timeouts": {
    "workflowExecTimeout": "PT1H"
}
```

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 3]
```diff
@@ -7,26 +7,37 @@
...
+It follows the format `PnDTnHnMn.nS` with days considered to be exactly 24 hours.
+For instance, `PT15M` configures 15 minutes, and `P2DT3H4M` defines 2 days, 3 hours and 4 minutes.
+
+== Workflow timeout 
+
+You can set up the maximum amount of time a workflow might be running before being cancelled. This is configured in the header section of the flow, by using link:{spec_doc_url}#workflowexectimeout-definition[WorkflowExecTimeout definition]. Only `duration` property is currently implemented. 
+Once cancelled, the workflow is considered finished and won't be accessible though a GET request anymore. So it behaves as interrupt was `true` by default. 
+
+For example, in order to cancel the workflow after an hour of execution, you might add the following snippet. 
+.Example of workflow timeout
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:24]:
```suggestion
For example, in order to cancel the workflow after an hour of execution, you might add the following snippet. 

.Example of workflow timeout
[source,json]
```

**Review by radtriste — COMMENTED** [review]
minor comments

**Review by radtriste — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Review by tiagodolphine — COMMENTED** [review]
LGTM, nice adding, the only improvement I think would be great, is to add a workflow example under https://github.com/kiegroup/kogito-examples/tree/main/serverless-workflow-examples/serverless-workflow-timeouts-showcase and reference it here similar to the other examples.

**Comment by fjtirado** [comment]
@tiagodolphine opened https://github.com/kiegroup/kogito-examples/pull/1547 to include the example on that project.

**Comment by tiagodolphine** [comment]
> @tiagodolphine opened [kiegroup/kogito-examples#1547](https://github.com/kiegroup/kogito-examples/pull/1547) to include the example on that project.

Thanks @fjtirado

**Review by tiagodolphine — APPROVED** [review]

**Review by wmedvede — APPROVED** [review]
LGTM

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 4]
```diff
@@ -7,26 +7,39 @@
 :jobs_service_image_url: https://quay.io/repository/kiegroup/kogito-jobs-service-allinone
 :jobs_service_image_usage_url: https://github.com/kiegroup/kogito-images#jobs-services-all-in-one
 
+
+Serverless Workflow specification defines a wide amount of possible link:{spec_doc_url}#workflow-timeout-definition[timeout configurations]. 
+Among them, {product_name} currently supports workflow and event timeouts. 
+
+Regardless its application scope (workflow or state), timeouts can not be defined as a specific point in time, but it should be an amount of time, a duration, which is considered to start when the referred scope becomes active. Timeouts uses link:https://en.wikipedia.org/wiki/ISO_8601[`ISO 8601` data and time standard] to specify a duration of time.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:14]:
```suggestion
Regardless of its application scope (workflow or state), you can not define the timeouts as a specific point in time, but they should be an amount of time, a duration, which is considered to start when the referred scope becomes active. Timeouts use the link:https://en.wikipedia.org/wiki/ISO_8601[`ISO 8601` data and time standard] to specify a duration of time.
```

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 5]
```diff
@@ -7,26 +7,39 @@
...
+It follows the format `PnDTnHnMn.nS` with days considered to be exactly 24 hours.
+For instance, `PT15M` configures 15 minutes, and `P2DT3H4M` defines 2 days, 3 hours and 4 minutes.
+
+== Workflow timeout 
+
+You can set up the maximum amount of time a workflow might be running before being cancelled. This is configured in the header section of the flow, by using link:{spec_doc_url}#workflowexectimeout-definition[WorkflowExecTimeout definition]. Only `duration` property is currently implemented.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:20]:
```suggestion
You can set up the maximum amount of time a workflow might be running before being canceled. This is configured in the header section of the flow, by using the link:{spec_doc_url}#workflowexectimeout-definition[WorkflowExecTimeout definition]. Only the `duration` property is currently implemented. 
```

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 6]
```diff
@@ -7,26 +7,39 @@
...
+Among them, {product_name} currently supports workflow and event timeouts. 
+
+Regardless its application scope (workflow or state), timeouts can not be defined as a specific point in time, but it should be an amount of time, a duration, which is considered to start when the referred scope becomes active. Timeouts uses link:https://en.wikipedia.org/wiki/ISO_8601[`ISO 8601` data and time standard] to specify a duration of time.
+It follows the format `PnDTnHnMn.nS` with days considered to be exactly 24 hours.
+For instance, `PT15M` configures 15 minutes, and `P2DT3H4M` defines 2 days, 3 hours and 4 minutes.
+
+== Workflow timeout 
+
+You can set up the maximum amount of time a workflow might be running before being cancelled. This is configured in the header section of the flow, by using link:{spec_doc_url}#workflowexectimeout-definition[WorkflowExecTimeout definition]. Only `duration` property is currently implemented. 
+Once cancelled, the workflow is considered finished and won't be accessible though a GET request anymore. So it behaves as interrupt was `true` by default.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:21]:
```suggestion
Once canceled, the workflow is considered to be finished and will not be accessible through a GET request anymore. So it behaves as if interrupt was `true` by default. 
```

**Review by kaldesai — APPROVED** [review]
@fjtirado I have added my suggestions otherwise your content looks good to me. thank you!

**Pull request merged**